src/commonTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/test_suite/SuiteUtils.kt

```diff
@@ -36,71 +36,71 @@ internal object SuiteUtils {
      *
      * @see deviationsInEvents
      */
-    val deviationsInResult: Set<YamlTestData.Id> = setOf(
+    val deviationsInResult: Set<Pair<YamlTestData.Id, String>> = setOf(
```

Comment:
I would recommend creating a data class with meaningful field names (e.g. `TestData(val id: ..., val label: String)`). It makes the intention more clear

Reply:
https://github.com/krzema12/snakeyaml-engine-kmp/pull/568